backend/backend.go

```diff
@@ -240,6 +242,15 @@ type Backend struct {
 
 	// isOnline indicates whether the backend is online, i.e. able to connect to the internet.
 	isOnline atomic.Bool
+
+	// quit is used to indicate to running goroutines that they should stop as the backend is being closed
+	quit chan struct{}
+
+	// enqueueUpdateForAccount is used to enqueue an update for an account.
+	enqueueUpdateForAccount chan *eth.Account
+
+	// updateETHAccountsCh is used to trigger an update of all ETH accounts.
+	updateETHAccountsCh chan struct{}
```

Comment:
I mentioned somewhere that we could wrap this into a dedicated struct and move all related methods there to pollute the backend less with ETH specifics. `ethUpdater *ETHUpdater` for example. Then unit tests could be added too more easily. Let's do this after this is merged?

Reply:
Yeah can do of course in a next PR to keep the scope smaller. I